infra/feast-operator/internal/controller/services/services.go

```diff
@@ -104,6 +124,11 @@ func (feast *FeastServices) removeFeastServiceByType(feastType FeastServiceType)
 	if err := feast.deleteOwnedFeastObj(feast.initFeastSvc(feastType)); err != nil {
 		return err
 	}
+	if _, shouldCreate := shouldCreatePvc(feast.FeatureStore, feastType); shouldCreate {
```

Comment:
we prob don't need the `shouldCreate` check here. this `removeFeastServiceByType` function indicates that everything related to that feast service type should be removed.